signac_dashboard/modules/image_viewer.py

```diff
@@ -38,22 +39,37 @@ def __init__(
         img_globs=("*.png", "*.jpg", "*.gif"),
         **kwargs,
     ):
-        super().__init__(name=name, context=context, template=template, **kwargs)
+        self._enabled_contexts = {"JobContext", "ProjectContext"}
+        super().__init__(
+            name=name,
+            context=context,
+            template=template,
+            **kwargs,
+        )
         self.img_globs = img_globs
 
-    def get_cards(self, job):
+    def get_cards(self, job_or_project):
+        if self.context == "JobContext":
+            jobid = job_or_project._id
```

Comment:
We recently refactored signac's `next` branch to use `job_id` instead of `jobid` everywhere. Just a note - we'd probably want to do the same in dashboard eventually.